tabs/tab_logit.py

```
@@ -274,66 +403,105 @@ def _cleanup_after_analysis():
         # Identify potential subgroups (2-10 levels)
         sg_cols = [c for c in cols if 2 <= d[c].nunique() <= 10]
-        
-        # Update Tab 1 Inputs
+
+        # Update Tab 1 (Binary Logit) Inputs
         ui.update_select("sel_outcome", choices=binary_cols)
         ui.update_selectize("sel_exclude", choices=cols)
         
-        # Update Tab 2 Inputs
+        # Generate interaction pair choices for Logit
+        interaction_choices = [f"{cols[i]} × {cols[j]}" for i in range(len(cols)) for j in range(i+1, len(cols)) if i != j]
```

Ruff (0.14.10)

String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)